src/astra_driver.cpp

```diff
@@ -717,6 +720,7 @@ sensor_msgs::msg::CameraInfo::SharedPtr AstraDriver::getDepthCameraInfo(int widt
 
 void AstraDriver::readConfigFromParameterServer()
 {
+  depth_frame_id_ = std::string("openni_depth_optical_frame");
```

Comment:
why use openni here ? this package doesn't depend on openni in any way does it ? Maybe "depth_optical_frame" or using "orbec" or "astra" prefix would be more appropriate